File: Haskell/temperatureconverter.hs
main :: IO ()
main = do
    putStrLn "Enter the temperature in Celsius:"
    input <- getLine
    let celsiusTemperature = read input :: Double

    let fahrenheitTemperature = (celsiusTemperature * 1.8) + 32

    putStrLn $ "The temperature in Fahrenheit is: " ++ show fahrenheitTemperature

File: C/temperatureconverter.c
#include <stdio.h>

int main() {
    double celsius, fahrenheit;

    printf("Enter the temperature in Celsius: ");
    scanf("%lf", &celsius);

    fahrenheit = (celsius * 1.8) + 32;

    printf("The temperature in Fahrenheit is: %.2lf\n", fahrenheit);

    return 0;
}